stripe-eventbridge-lambda/README.md

```diff
@@ -0,0 +1,115 @@
+    ```
+    cdk deploy --parameters stripeEventBusName=SAAS_EVENT_BUS_NAME_HERE
+    ```
+
+4. Note the outputs from the CDK deployment process. These contain the resource names and/or ARNs which are used for testing. This stack will output the name of the Lambda function deployed for testing to the CLI. See the example below.
+
+```
+Outputs:
+StripeIntegrationStack.StripeProcessFailedPaymentLambdaOutput = StripeIntegrationStack-StripeProcessFailedPaymentL-nIPhn7sPQx9e
+```
+
+## How it works
+
+This service interaction uses an existing Stripe SaaS integration in the customer's AWS account. If you do not have the Stripe SaaS integration set up in your AWS account, please set it up before deploying this pattern. View the [integration on the Stripe Docs](https://docs.stripe.com/event-destinations/eventbridge). See [Stripe's documentation](https://stripe.com/docs/api/events/types) for available events.
+
+This pattern demonstrates how to:
+1. Write EventBridge rules that match Stripe's event pattern
+2. Send events from Stripe's EventBridge SaaS integration to Amazon CloudWatch for logging and debugging
+3. Transform Stripe events using AWS Lambda
+4. Store failed payment details in a DynamoDB table
+
+See the below architecture diagram from the data flow of this pattern.
+
+![Architecture Diagram](./img/readme-arch-diagram.png)
+
+## Testing
+
+### Test the AWS Lambda Function
+
+The event.json file included in this pattern is a sample EventBridge event from Stripe. This event can be used to test the Lambda function and EventBridge rules deployed by this pattern.
+
+To test the Lambda function via the CLI, copy and paste the following command, replacing the variables in <> with your own values:
+```
+aws lambda invoke --function-name <StripeIntegrationStack-StripeProcessFailedPaymentL-nIPhn7sPQx9e> --payload file://event.json response.json
+```
+
+You should receive a response that looks like:
+```
+{
+    "StatusCode": 200,
+    "ExecutedVersion": "$LATEST"
+}
+```
+
+The command should have create a response.json file in your directory. If you open this file, you see the output of the Lambda function.
+
+### Test the EventBridge Rule
+
+You cannot put events on the Stripe SaaS event bus, only Stripe can publish events to this event bus. To test that the EventBridge rules deployed by this pattern were successfully deployed, follow these instructions:
+
+1. Navigate to the Amazon EventBridge console. Select "Rules".
+
+2. From the Event bus list, choose the SaaS event bus associated with your Stripe partner event source.
+
+3. From the Rules list, select the "StripeIntegrationStack-StripeFailedPaymentEventsRul-cSTSp1wtPkkB"
```

Comment:
The suffix is randomly generated and will not match the users resources. It's probably better to word this differently